spec.bs

```diff
@@ -369,19 +739,70 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 <div algorithm>
   The <dfn method for=Fence>reportEvent(<var ignore>event</var>)</dfn> method steps are:
 
-  1. <span class=XXX>Fill this out!</span>
+  1. Let |instance| be [=this=]'s [=relevant global object=]'s [=associated Document=]'s [=node
+     navigable=]'s [=navigable/traversable navigable=]'s [=fenced frame config instance=].
+
+  1. If |instance| is null, then return.
+
+  1. If |instance|'s [=fenced frame config instance/mapped url=] is null, then the behavior is
+     unspecified.
```

Comment:
This is a little bit strange to me. What does our implementation do? Surely we don't just defer to C++ undefined behavior right? Can we bail out here? (same with the instances below).

Reply:
We have a CHECK because this case is currently unreachable. We could make [=mapped url=] not nullable for now, but then we might have to change it later.

Reply:
I'll make this a spec assert, but if that's really a requirement for now then we should just make the member non-nullable and change that in the future if we ever support that.